xtext-website/documentation/350_continuous_integration.md

```diff
@@ -247,7 +247,7 @@ To further speed up the p2 dependency resolution step, use the concrete build re
 
 | Xtext         | EMF           | MWE2/MWE    | Xpand       | Eclipse     | All included in |
 | ------------- | ------------- | ----------- | ----------- | ----------- | ----------- |
-| [2.22.0]({{page.upsite.xtext}}releases/2.22.0/)           | [2.22.0]({{page.upsite.eclipse}}modeling/emf/emf/builds/release/2.22) (2.20.0)     | [2.11.2]({{page.upsite.mwe}}releases/2.11.2/) (2.9.1) | [2.2.0]({{page.upsite.xpand}}releases/R201605260315) (1.4)  | [4.16.0]({{page.upsite.eclipse}}releases/2020-06) (4.7.3) | [2020-06]({{page.upsite.eclipse}}releases/2020-06)|
+| [2.22.0]({{page.upsite.xtext}}releases/2.22.0/)           | [2.22.0]({{page.upsite.eclipse}}modeling/emf/emf/builds/release/2.22) (2.20.0)     | [2.11.3]({{page.upsite.mwe}}releases/2.11.3/) (2.9.1) | [2.2.0]({{page.upsite.xpand}}releases/R201605260315) (1.4)  | [4.16.0]({{page.upsite.eclipse}}releases/2020-06) (4.7.3) | [2020-06]({{page.upsite.eclipse}}releases/2020-06)|
```

Comment:
`../2.11.3/) (2.9.1)  ` <- should this be updated to 2.11.3, too?

Reply:
this is the minmal version and not updated.
never was
we should talk separatedly on monday